### ISSUE TITLE
New graphic design

### ISSUE THREAD

**Issue by dannyx93** [ihofmann/open-websoccer#92]
The basic theme is outdated. Anyone planning to do a theme update?

**Comment by Requiao** [comment]
Yes, it is quite outdated. If you have any ideas, we can start working together to improve the visual aspect of the site. In vManager.net I made some changes based on themes from other projects.

**Comment by dannyx93** [comment]
I was thinking to customize one of the Bootstrap 4 Admin Panel themes. It's just a matter of rewriting the Twig template engine to a new look.

**Comment by ceriyvolck** [comment]
@Requiao can you share your friendlies? please) https://vmanager.net/?page=friendly-schedule

**Comment by ceriyvolck** [comment]
I have a revised version with different skills and normal tabs. But there are no friendly matches...
[https://ibb.co/Mk88WF4](url)

**Comment by ceriyvolck** [comment]
https://ibb.co/SNjVykz

**Comment by dannyx93** [comment]
will you share the template?

**Comment by dannyx93** [comment]
I can see on your template the box from my template.

**Comment by ceriyvolck** [comment]
@dannyx93 based on what you've done, I'll be able to post it on Monday. there is no access now.

**Comment by ceriyvolck** [comment]
@dannyx93 как вы подключили товарищеские матчи? twig есть, но нет для него класса, что бы прописать в шаблонизатор. Пытался переделать из молодёжной команды, но не получилось... Зато запустил тренировки навыков каждый день, в зависимости от потенциала игрока (потенциал скртыт как футбал менеджер). есть переработанные показатели, что бы не были так часто большие счета... всем эти поделюсь на следующей неделе. напишу тогда здесь. надеюсь в ответ кто то поделиться товарищескими матчами...
how did you enable friendlies? there is a twig, but there is no class for it to write in the template engine. I tried to change it from the youth team, but it didn't work out... But I started training skills every day, depending on the potential of the player (the potential of skryt as a football Manager). there are revised indicators that would not be so often large bills... I'll share all of these next week. I'll write here then. I hope the answer is someone to share in friendly matches...

**Comment by ztxjack** [comment]
Hi guys, could you leave any emails for contact? I wanna find someone to do the teamwork on the improvement of engine and graphics if you have such interest. @Requiao @dannyx93 @ceriyvolck

**Comment by ceriyvolck** [comment]
@ztxjack ceriy.volck@yandex.ru

**Comment by joseborges** [comment]
https://discord.gg/HCuGt7h

**Comment by ihofmann** [comment]
I agree, feel free to open a ticket with specific ideas.
